PAM_app/app/src/main/java/com/example/pam_app/foodAndDrinks/RecyclerActivity.kt

```diff
@@ -0,0 +1,101 @@
+package com.example.pam_app.foodAndDrinks
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.DefaultItemAnimator
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.pam_app.R
+
+
+class RecyclerActivity : AppCompatActivity() {
+    private val foodList = ArrayList<FoodModel>()
+    private lateinit var foodAdapter: FoodAdapter
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_recycler)
```

Comment:
Change init of activity view to android view binding https://developer.android.com/topic/libraries/view-binding